devsetup/standalone/openstack.sh

```diff
@@ -92,7 +93,108 @@ if [ "$EDPM_COMPUTE_CEPH_ENABLED" = "true" ] ; then
     CEPH_ARGS=${CEPH_ARGS:-"-e ~/deployed_ceph.yaml -e /usr/share/openstack-tripleo-heat-templates/environments/cephadm/cephadm-rbd-only.yaml"}
     ENV_ARGS+=" ${CEPH_ARGS}"
 fi
+
+# NOTE: For other nodes than node0, deploy it as extra Heat stacks,
+# by using env files prepared after the standalone node0.
+# See https://docs.openstack.org/project-deploy-guide/tripleo-docs/wallaby/deployment/standalone.html#deploy-the-remote-compute-node
+EDGE="$HOME/edge${EDPM_COMPUTE_SUFFIX}"
+if [[ "$EDPM_COMPUTE_SUFFIX" != "0"  ]]; then
+    sed -ir "s/edge0/edge${EDPM_COMPUTE_SUFFIX}/g" ${EDGE}_services.yaml
+    ENV_ARGS+=" -e ${EDGE}_services.yaml"
+    ENV_ARGS+=" -e ${EDGE}_endpoint-map.json"
+    ENV_ARGS+=" -e ${EDGE}_net-ip-map.json"
+    ENV_ARGS+=" -e ${EDGE}_all-nodes-extra-map-data.json"
+    ENV_ARGS+=" -e ${EDGE}_extra-host-file-entries.json"
+    ENV_ARGS+=" -e ${EDGE}_oslo.json"
+fi
 ENV_ARGS+=" -e $HOME/containers-prepare-parameters.yaml"
 ENV_ARGS+=" -e $HOME/deployed_network.yaml"
 
 sudo ${CMD} ${CMD_ARGS} ${ENV_ARGS}
+
+# NOTE: after node0 deployed, extract env files for multi-stack standalone deployments
+if [[ "$EDPM_COMPUTE_SUFFIX" == "0"  ]]; then
+    unset OS_CLOUD
+    export OS_AUTH_TYPE=none
+    export OS_ENDPOINT=http://127.0.0.1:8006/v1/admin
+    export OS_CLOUDNAME=heat
+    STANDALONE_LATEST=$(find ~/standalone-ansible-* -type d -printf "%T@ %p\n" | sort -n | cut -d' ' -f 2- | tail -n 1)
+    set +e
+    openstack stack output show standalone EndpointMap --format json
+    if [ $? -ne 0 ]; then
+        set -e
+        HEAT_LATEST=$(find ~/heat_launcher* -type d -printf "%T@ %p\n" | sort -n | cut -d' ' -f 2- | tail -n 1)
+        pkill -f /usr/bin/heat-all
+        openstack tripleo launch heat --restore-db --heat-dir="$HEAT_LATEST" --heat-type=native
+        cd $HEAT_LATEST
+    fi
+
+    openstack stack output show standalone EndpointMap --format json \
+    | jq '{"parameter_defaults": {"EndpointMapOverride": .output_value}}' \
+    > ${EDGE}_endpoint-map.json
+
+    openstack stack output show standalone EndpointMap --format json \
+    | jq '{"parameter_defaults": {"RoleNetIpMap": .output_value}}' \
+    > ${EDGE}_net-ip-map.json
```

Comment:
not sure if this is needed, if so, we should update the tripleo docs as well

https://docs.openstack.org/project-deploy-guide/tripleo-docs/wallaby/deployment/standalone.html#extract-deployment-information-from-the-controller-node